package/crds/provider-ceph.ceph.crossplane.io_buckets.yaml

```diff
@@ -51,6 +51,15 @@ spec:
           spec:
             description: A BucketSpec defines the desired state of a Bucket.
             properties:
+              autoPause:
```

Comment:
I am curious, why would we ever want to stop reconciling a bucket?

Reply:
Anybody who wants unlimited bucket without unlimited resources (mem, cpu, net)